assets/css/custom.css

```
@@ -87,14 +798,17 @@
 
 .font-h1 {
   @apply font-headline font-medium text-3xl md:text-4xl leading-tight;
+  text-wrap: balance;
```

Stylelint (17.13.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)